prow/slack/client.go

```diff
@@ -27,6 +27,7 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+//Logger : TODO (alisondy) Write documentation
```

Comment:
I don't think this kind of stub is helpful

Reply:
This masks the lint error by pretending to have fixed the lint, without actually fixing the lint :(

Reply:
Understandable ✨